source/code/include/playfab/PlayFabCallRequestContainerBase.h

```diff
@@ -6,7 +6,7 @@
 namespace PlayFab
 {
     class CallRequestContainerBase;
-    typedef std::function<void(int, std::string, std::shared_ptr<CallRequestContainerBase>)> CallRequestContainerCallback;
+    typedef std::function<void(int, std::string, const std::shared_ptr<CallRequestContainerBase>&)> CallRequestContainerCallback;
```

Comment:
Since the callback is public facing AND we do want users to be granted the ability of transferring ownership. I can also agree with your disagreement here.

Reply:
Since this would be a public breaking change (users of any classes that derive this base class would need to change to a const ref param), I can get behind not taking this one.